### ISSUE TITLE
Accounts declared in the flow.json are not created when emulator is launched 

### ISSUE THREAD

**Issue by bluesign** [onflow/flow-cli#1109]
https://github.com/onflow/flow-emulator/issues/405

from flow-emulator repo by @cybercent 

###Instructions

flow accounts create adds and account to the flow.json file.

When relaunching the emulator the accounts defined in the flow.json are not recreated.

There should be a command to recreate the accounts defined in the flow.json file.

###Problem

Accounts defined in flow.json not present in emulator.

Steps to Reproduce

{
	"accounts": {
		"AccountA": {
			"address": "01cf0e2f2f715450",
			"key": "some_private_key"
		},
		"AccountB": {
			"address": "179b6b1cb6755e31",
			"key": "some_private_key"
		},
		"AccountC": {
			"address": "f3fcd2c1a78f5eee",
			"key": "some_private_key"
		},
		"default": {
			"address": "01cf0e2f2f715450",
			"key": "some_private_key"
		},
		"emulator-account": {
			"address": "f8d6e0586b0a20c7",
			"key": "some_private_key"
		}
	},
	"deployments": {
		"emulator": {
			"default": [],
			"emulator-account": []
		}
	}
}

###Acceptance Criteria

There should be a command to recreate the accounts defined in the flow.json file.

**Comment by chasefleming** [comment]
@bluesign This feels somewhat like what `flow dev` in [super commands](https://developers.flow.com/tooling/flow-cli/super-commands#develop) is trying to solve, no? The difference here would be that it would be looking to your folder structure to create accounts on startup and not your flow.json. I still think `flow dev` needs some work though, so maybe we should iron issues out on it if it's not yet satisfying what you need. Thoughts?

**Comment by bluesign** [comment]
yeah I agree `flow dev` is way to go, I just moved the issue from emulator repo ( it is originally from @cybercent ) 

Unfortunately on emulator side we are not using flow.json, so my hands are a bit tied there.
